Solobarine.github.io/style.css

```diff
@@ -831,4 +842,63 @@ header {
     justify-content: center;
     margin: 25px 0;
   }
+
+  .pop-menu {
+    width: auto;
+    background: #1c1a19;
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  .pop {
+    display: none;
+  }
+
+  #close {
+    float: right;
+    margin-top: 10px;
+    margin-right: 10px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .img-button {
+    width: 100%;
+    height: 100%;
+  }
+
+  .nav-tools {
+    font-family: 'Inter', sans-serif;
+    font-size: 22px;
+    line-height: 44px;
+    color: #fff5e1;
+    list-style: none;
+    width: 270px;
+    margin-top: 10px;
+    border-bottom: 1px solid #ccc;
+    overflow-y: hidden;
+  }
+
+  .first-tool {
+    padding-top: 20px;
+  }
+
+  li a:link {
+    text-decoration: none;
+  }
+
+  .navigation {
+    background: #1c1a19;
+    padding: 20px;
+  }
```

Comment:
Great job team 🤝

- [ ] [Optional] As specified in the project requirements, we are supposed to display only the popup when we click on the hamburger icon. However, I can see from your design we can easily scroll through the rest of your design. Try to maintain just the popup whenever you click on the hamburger icon.
![scroll](https://user-images.githubusercontent.com/63917725/180595648-726482ef-bbcc-47c8-a722-728afac2f412.PNG)